.github/workflows/posix-deps-apt.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
+
+# Workaround missing on ubuntu 24.04 libmpdec-dev
+sudo add-apt-repository ppa:ondrej/php
```

Comment:
Would it be useful to backport this change to 3.13 and 3.14 branches? Which libmpdec is used on these branches? The vendored copy or the system library? If it makes sense to backport, I would suggest to extract these changes (.github/workflows/posix-deps-apt.sh) into a separated PR.

Reply:
Ok, let's keeep the bundled copy in 3.13 and 3.14.